modules/serverless-event-delivery-component-behaviors.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-event-delivery-component-behaviors_{context}"]
+= Event delivery behavior for Knative Eventing components
+
+Different Knative Eventing components have their own behavior patterns that are followed for event delivery.
```

Comment:
It's not clear to me what a 'behavior pattern' is.  Looking at the subchapter it follows I think this should describe that there a loosely coupled topics that are common to all eventing components. So maybe `shared behavior` is a better wording in this context ?

Reply:
Did some rewording / editing here, LMK what you think 🙂